packages/ui-extensions/src/surfaces/admin/extension-points.ts

```diff
@@ -14,6 +14,46 @@ export interface ExtensionPoints {
     CustomerSegmentationTemplateApi<'admin.customers.segmentation-templates.render'>,
     CustomerSegmentationTemplateComponent
   >;
+
+  // Blocks
+  'admin.product-details.block.render': RenderExtension<
+    StandardApi<'admin.product-details.block.render'>,
+    AnyComponent
+  >;
+  'admin.order-details.block.render': RenderExtension<
+    StandardApi<'admin.order-details.block.render'>,
+    AnyComponent
+  >;
+  'admin.customer-details.block.render': RenderExtension<
+    StandardApi<'admin.customer-details.block.render'>,
+    AnyComponent
+  >;
+
+  // Actions
+  'admin.product-details.action.render': RenderExtension<
+    StandardApi<'Playground'>,
+    AnyComponent
+  >;
+  'admin.order-details.action.render': RenderExtension<
+    StandardApi<'Playground'>,
+    AnyComponent
+  >;
+  'admin.customer-details.action.render': RenderExtension<
+    StandardApi<'Playground'>,
+    AnyComponent
+  >;
+  'admin.product-index.action.render': RenderExtension<
+    StandardApi<'Playground'>,
+    AnyComponent
+  >;
+  'admin.order-index.action.render': RenderExtension<
+    StandardApi<'Playground'>,
+    AnyComponent
+  >;
+  'admin.customer-index.action.render': RenderExtension<
+    StandardApi<'Playground'>,
+    AnyComponent
+  >;
```

Comment:
I think we're missing the bulk action extension points for action extensions. I'm not sure what their names should be because they're also missing from the spreadsheet

Reply:
Good catch!